.luacheckrc

```diff
@@ -77,7 +77,7 @@ files["archtec_matterbridge/rx.lua"] = {
 }
 
 files["archtec/scripts/common.lua"] = {
-	globals = {"core.kick_player", "core.disconnect_player", "minetest.registered_chatcommands.?.mod_origin"},
+	globals = {"core.kick_player", "core.kick_inactive_player", "core.disconnect_player", "minetest.registered_chatcommands.?.mod_origin"},
```

Comment:
Feels dirty to pollute supposed "globals", but it's the only way this can work if I'm to use a separate function for inactive kick.